.pipelines/build.cmd

```diff
@@ -33,9 +33,30 @@ echo Copying contents...
 cd /D "%~dp0"
 if errorlevel 1 goto :error
 
-xcopy /e /i c:\webrtc\src\out ..\output
-if errorlevel 1 goto :error
+call :copyFiles c:\webrtc\src\out\msvc\uwp\Release\x64\obj\webrtc.lib ..\output\msvc\uwp\Release\x64\obj\
+call :copyFiles c:\webrtc\src\api\*.h ..\include\api\
+call :copyFiles c:\webrtc\src\audio\*.h ..\include\audio\
+call :copyFiles c:\webrtc\src\base\*.h ..\include\base\
+call :copyFiles c:\webrtc\src\call\*.h ..\include\call\
+call :copyFiles c:\webrtc\src\common_audio\*.h ..\include\common_audio\
+call :copyFiles c:\webrtc\src\common_video\*.h ..\include\common_video\
+call :copyFiles c:\webrtc\src\logging\*.h ..\include\logging\
+call :copyFiles c:\webrtc\src\media\*.h ..\include\media\
+call :copyFiles c:\webrtc\src\modules\*.h ..\include\modules\
+call :copyFiles c:\webrtc\src\p2p\*.h ..\include\p2p\
+call :copyFiles c:\webrtc\src\pc\*.h ..\include\pc\
+call :copyFiles c:\webrtc\src\rtc_base\*.h ..\include\rtc_base\
+call :copyFiles c:\webrtc\src\rtc_tools\*.h ..\include\rtc_tools\
+call :copyFiles c:\webrtc\src\stats\*.h ..\include\stats\
+call :copyFiles c:\webrtc\src\system_wrappers\*.h ..\include\system_wrappers\
+call :copyFiles c:\webrtc\src\video\*.h ..\include\video\
+call :copyFiles c:\webrtc\src\common_types.h ..\include\
```

Comment:
Please add abseil and libyuv:

xcopy /s /i c:\webrtc\src\third_party\abseil-cpp\absl\*.h C:\WinRTC_GitHub\MyFirstWinRtc\packages\Microsoft.Win
RTC.libwebrtc.84.0.12980001-alpha\build\native\include\absl\

xcopy /s /i c:\webrtc\src\third_party\libyuv\include\*.h C:\WinRTC_GitHub\MyFirstWinRtc\packages\Microsoft.
WinRTC.libwebrtc.84.0.12980001-alpha\build\native\include\